edb/server/protocol/binary.pyx

```diff
@@ -1193,6 +1193,8 @@ cdef class EdgeConnection:
         return query_unit
 
     def signal_side_effects(self, side_effects):
+        if not self.server._accept_new_tasks:
```

Comment:
This should have some "public" method or property. It's fine to use underscored attribute in `server.py` but not here.